README.md

```diff
@@ -1,25 +1,37 @@
-# Alchemy React Base Template
+# Kickball Plan
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Components
 
-Use this template for all your "from scratch" deliverables. To start, simply run
+-Header with NAV Links to Home, Teams, Players OR add Nav links on each page
 
-- `npm install`
-- `npm start`
+## Views
 
-## Available Scripts
+Home
+Teams --Nav Link and Router
+Players --Nav Link and Router
 
-In the project directory, you can run:
+## Team View
 
-### `npm start`
+-Lists Teams with details on team and players on the team
+-fetch teams
+-fetch team by id (use supabase client)
+-have a teamdetails.js using params.id return player details
+-add in Router
+-add Nav Link back to Home???
```

Comment:
Love your thought process and planning in the READme, super important for building these intricate apps.